tests/InputFiles/Poisson/Input3D.yaml

```diff
@@ -0,0 +1,25 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: SolvePoissonProblem3D
+# Check: execute
+
+AnalyticSolution:
+    WaveNumbers: [1, 1, 1]
+
+DomainCreator:
+  Brick:
+    LowerBound: [0, 0, 0]
+    UpperBound: [3.141592653589793, 3.141592653589793, 3.141592653589793]
+    IsPeriodicIn: [false, false, false]
+    InitialRefinement: [1, 1, 1]
+    InitialGridPoints: [2, 2, 2]
+
+NumericalFluxParams:
+    PenaltyParameter: 5.729577951308232
```

Comment:
where does this value come from?

Reply:
It's p^2/h = 18/pi for 1D and 2D, but p is different here.
@nilsleiffischer Should the 3D case be 8/pi instead?

Might be worth a comment in either case.

Reply:
Sure I added a comment. This is also documented in `Poisson::FirstOrderInternalPenaltyFlux`. Corrected this also for p=2 in the 3D file.